docs/content/docs/user-docs/ack-tags.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Managing ACK Default Tags"
```

Comment:
I'm tempted to make this more generic... something like "Managing Tags on your AWS Resources"